group.yml

```
@@ -1,4 +1,4 @@
-freeze_automation: weekdays
+freeze_automation: true
```

normally we'd just
```suggestion
freeze_automation: scheduled
```
`true` is a hard freeze, you can't even run builds manually.

i guess that's fine for 4.9, just calling it out.